Build Failed for latest commit

### System

OS: Linux/ fedora-34
Wayland 
cargo version - cargo 1.54.0-nightly (0cecbd673 2021-06-01)
commad ran -  cargo build --release --verbose
### Logs
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-Wl,--as-needed" "-L" "/home/abhi/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/home/abhi/extras/alacritty/target/release/deps/alacritty-368205aaf3cb8dc1.alacritty.709c42f794df0aaf-cgu.7.rcgu.o" "-o" "/home/abhi/extras/alacritty/target/release/deps/alacritty-368205aaf3cb8dc1" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-L" "/home/abhi/extras/alacritty/target/release/deps" "-L" "/usr/lib64" "-L" "/usr/lib64" "-L" "/usr/lib64" "-L" "/home/abhi/extras/alacritty/target/release/build/expat-sys-7e968dc241d043a7/out/lib" "-L" "/home/abhi/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/tmp/rustclzoSWY/libexpat_sys-7b3b47c6cd9173a2.rlib" "-Wl,--start-group" "-Wl,--end-group" "/home/abhi/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-d231748acf961207.rlib" "-Wl,-Bdynamic" "-lfreetype" "-lfontconfig" "-lfreetype" "-lfreetype" "-lxcb" "-lxcb-render" "-lxcb-shape" "-lxcb-xfixes" "-lxkbcommon" "-ldl" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc"
  = note: /usr/bin/ld: cannot find -lxkbcommon
          collect2: error: ld returned 1 exit status

Caused by:
  process didn't exit successfully: `rustc --crate-name alacritty --edition=2018 alacritty/src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=3 -C lto -C debuginfo=1 --cfg 'feature="default"' --cfg 'feature="png"' --cfg 'feature="wayland"' --cfg 'feature="wayland-client"' --cfg 'feature="x11"' --cfg 'feature="x11-dl"' -C metadata=368205aaf3cb8dc1 -C extra-filename=-368205aaf3cb8dc1 --out-dir /home/abhi/extras/alacritty/target/release/deps -L dependency=/home/abhi/extras/alacritty/target/release/deps --extern alacritty_config_derive=/home/abhi/extras/alacritty/target/release/deps/libalacritty_config_derive-89519b92df98ef24.so --extern alacritty_terminal=/home/abhi/extras/alacritty/target/release/deps/libalacritty_terminal-64aaef7041630cf1.rlib --extern bitflags=/home/abhi/extras/alacritty/target/release/deps/libbitflags-f766c4bd06771efd.rlib --extern clap=/home/abhi/extras/alacritty/target/release/deps/libclap-c3bef305e500f805.rlib --extern copypasta=/home/abhi/extras/alacritty/target/release/deps/libcopypasta-663f59ab734aef5c.rlib --extern crossfont=/home/abhi/extras/alacritty/target/release/deps/libcrossfont-8fea86b105c4101c.rlib --extern dirs=/home/abhi/extras/alacritty/target/release/deps/libdirs-562187c126dd5240.rlib --extern fnv=/home/abhi/extras/alacritty/target/release/deps/libfnv-5297702bd12e7d0f.rlib --extern glutin=/home/abhi/extras/alacritty/target/release/deps/libglutin-1c179ad8405f7e51.rlib --extern libc=/home/abhi/extras/alacritty/target/release/deps/liblibc-a733b587b74dc59c.rlib --extern log=/home/abhi/extras/alacritty/target/release/deps/liblog-2745c494cf17a688.rlib --extern notify=/home/abhi/extras/alacritty/target/release/deps/libnotify-d99801e16c11d562.rlib --extern parking_lot=/home/abhi/extras/alacritty/target/release/deps/libparking_lot-ef89f59c483f9cd8.rlib --extern png=/home/abhi/extras/alacritty/target/release/deps/libpng-12bdd105c0137364.rlib --extern serde=/home/abhi/extras/alacritty/target/release/deps/libserde-6e2d468d1a1f418f.rlib --extern serde_json=/home/abhi/extras/alacritty/target/release/deps/libserde_json-c222a0531ef21541.rlib --extern serde_yaml=/home/abhi/extras/alacritty/target/release/deps/libserde_yaml-20f7835c848362d7.rlib --extern time=/home/abhi/extras/alacritty/target/release/deps/libtime-1fbcdc72f69d23c2.rlib --extern unicode_width=/home/abhi/extras/alacritty/target/release/deps/libunicode_width-5bcfa79d7258eeb6.rlib --extern wayland_client=/home/abhi/extras/alacritty/target/release/deps/libwayland_client-3f9b6eadfd63be27.rlib --extern x11_dl=/home/abhi/extras/alacritty/target/release/deps/libx11_dl-d8ebffdec29ed09a.rlib --extern xdg=/home/abhi/extras/alacritty/target/release/deps/libxdg-627cc6a3f788f482.rlib -L native=/usr/lib64 -L native=/usr/lib64 -L native=/usr/lib64 -L native=/home/abhi/extras/alacritty/target/release/build/expat-sys-7e968dc241d043a7/out/lib` (exit status: 1)

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -91,7 +91,7 @@ On Arch Linux, you need a few extra libraries to build Alacritty. Here's a
 to be missing, please open an issue.
 
 ```sh
-pacman -S cmake freetype2 fontconfig pkg-config make libxcb
+pacman -S cmake freetype2 fontconfig pkg-config make libxcb libxkbcommon
 ```
 
 #### Fedora
@@ -101,7 +101,7 @@ command that should install all of them. If something is still found to be
 missing, please open an issue.
 
 ```sh
-dnf install cmake freetype-devel fontconfig-devel libxcb-devel g++
+dnf install cmake freetype-devel fontconfig-devel libxcb-devel libxkbcommon-devel g++
 ```
 
 #### CentOS/RHEL 7
@@ -111,7 +111,7 @@ command that should install all of them. If something is still found to be
 missing, please open an issue.
 
 ```sh
-yum install cmake freetype-devel fontconfig-devel libxcb-devel xcb-util-devel
+yum install cmake freetype-devel fontconfig-devel libxcb-devel libxkbcommon-devel xcb-util-devel
 yum group install "Development Tools"
 ```
 

